kolibri/utils/cli.py

```diff
@@ -232,13 +259,12 @@ def __init__(self, *args, **kwargs):
         super(KolibriDjangoCommand, self).__init__(*args, **kwargs)
 
     def invoke(self, ctx):
+        """
+        Initialize Kolibri and run sanity checks.
+        """
         # Check if the current user is the kolibri user when running kolibri from Debian installer.
```

Comment:
```suggestion
```

Reply:
Thanks for spotting that, fixed and pushed